src/containers/offer-page/chat-dialog-window/ChatDialogWindow.tsx

```diff
@@ -36,16 +36,37 @@ interface ChatDialogWindow {
 
 const ChatDialogWindow: FC<ChatDialogWindow> = ({ chatInfo }) => {
   const [textAreaValue, setTextAreaValue] = useState<string>('')
+  const [isInitSended, setIsInitSended] = useState<boolean>(false)
+  const [isRedirected, setIsRedirected] = useState<boolean>(false)
   const { setAlert } = useSnackBarContext()
   const { t } = useTranslation()
   const scrollRef = useRef<HTMLDivElement | null>(null)
   const { setChatInfo } = useChatContext()
 
+  const handleErrorResponse = useCallback(
+    (errorResponse: ErrorResponse) => {
+      setAlert({
+        severity: snackbarVariants.error,
+        message: errorResponse ? `${errorResponse.message}` : ''
+      })
+    },
+    [setAlert]
+  )
+
   const getMessages = useCallback(
-    () => messageService.getMessages({ chatId: chatInfo.chatId }),
+    () => messageService.getMessages({ chatId: chatInfo.chatId as string }),
```

Comment:
does you really need to use `as string` here if it's already typed?

Reply:
> does you really need to use `as string` here if it's already typed?

It turns out that it needs it. `.getMessages` wants to receive chatId as a string, but `chatInfo.chatId` can be `string | Pick<ChatResponse, "_id">`, so I just cast it to a `string` type.